### PR TITLE
feat(accounts): created a new getting started with accounts guide

### PR DESCRIPTION
**Pull request by homelessbirds**

### CODE REVIEW

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+* Maximize data ingest
+* Keep data organized 
+* Monitor multiple environments, like staging and development
+* Manage billing across different areas of your organization
+
+This doc will guide you through setting up your account structure. Creating accounts is a good thing to do before you begin ingesting data. It can be harder to make changes after you connect New Relic to your applications because you risk losing historical context.
+
+<Callout variant="tip">
+If you're using Free or Standard editions, you only have one account available and you won't need this doc but feel free to revisit if you decide to change to [Pro or Enterprise](https://newrelic.com/pricing).
```
Comment (akristen) [inline src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:28]: I'd remove this `.` here because it formats a bit odd in the callout (in its own line)

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+</Callout>
+
+## Before you get started [#begin]
+
+In this guide we're going to help you create and manage your New Relic account. Before you get started we want to make sure you're familiar with our terms:
+
+* **Organization**:  an organization represents a New Relic customer. Your New Relic organization contains everything relevant to you and your team members: your organization's accounts, users, and data.
```
Comment (akristen) [inline src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:35]: I might clarify that an organization is the broadest or most basic unit? Like an org contains all of the smaller units of ~user?

Comment (akristen) [reply src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:35]: Also, inconsistent capitalization after the colon in these three definitions. "an" should be "An"

Comment (akristen) [reply src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:35]: Hm, it's unclear to me still what an account is exactly. I always thought that organizations can have multiple accounts within them for different teams, and those accounts have their own data sources? Is there any way we can disambiguate this a bit more?

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+* **Account**: When you sign up for New Relic your organization is automatically assigned one account. An account can have many sources of data reporting to it. For example, a single account could have data reporting from our infrastructure agent, an APM agent, and other integrations. All data reported to New Relic requires an account ID, which tells New Relic which account that data belongs in. That ID is also used for some account-specific tasks, like making API calls.
+* **User**: In your organization, your New Relic users are granted access to specific accounts that are relevant to their duties and responsibilities. To manage users' access to accounts, you grant groups access to specific roles on specific accounts. Our user management system allows you to create the user access you need, whether that's a relatively simple setup with just a few roles across a few accounts, or a complex one with many roles across many accounts. Learn more about user management.
+
+## What account structure is right for me? [#account-structure]
```
Comment (akristen) [inline src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:39]: this more or less answers my question from 'Accounts' above. I might foreshadow in the above bullet point that how account(s) are structured within an org can vary across different orgs who might have diff needs

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+Signing up for New Relic automatically generates an organization. An organization contains one account unless you manually create more. One account can be perfect for many companies and it might not be necessary for you to add more. 
+
+Some indications that one account is right for you:
+
+* You have 200 employees or less and you don't anticipate growing too much
+* You aren't ingesting vast amounts of data. If you're not sure, use [our spreadsheet](https://newrelic.com/blog/nerdlog/estimate-data-cost) to estimate the amount of data you might send to New Relic
+* You expect to send a moderate amount of alerts. If you're not sure, review our [alerting rules and limits](/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/rules-limits-alerts/)
+
+If these don't apply to you, it might be good to consider a multiple account structure.
```
Comment (akristen) [inline src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:49]: I might rephrase this to not be negative. 'If these apply to you, then you only need one account. Otherwise, we recommend considering a multi-account structure to fit your needs"

Comment (akristen) [reply src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:49]: also is it one or all need to apply? If yoou have 200 employees or less but you're still consuming a lot of data, would it be better to be multiaccount? Just something to consider

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+If you create multiple accounts you can observe data across multiple environments. With multiple accounts, you can monitor changes before they go to production. If you’re updating your application weekly, or even daily, it is important that you have a strong testing environment in place so you can make sure each deployment is smooth and error free.
+
+
+<img
+    title="multiple environments"
+    alt="a diagram demonstrating how to use New Relic with multiple environments"
+    src={accountsMultipleAccountEnvironments}
+/>
+
+### Use multiple accounts to separate projects with unique data sets [#data-sets]
```
Comment (akristen) [inline src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:62]: Potentially stronger title: Separate projects with unique data sets with multiple accounts

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+* You have 200 employees or less and you don't anticipate growing too much
+* You aren't ingesting vast amounts of data. If you're not sure, use [our spreadsheet](https://newrelic.com/blog/nerdlog/estimate-data-cost) to estimate the amount of data you might send to New Relic
+* You expect to send a moderate amount of alerts. If you're not sure, review our [alerting rules and limits](/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/rules-limits-alerts/)
+
+If these don't apply to you, it might be good to consider a multiple account structure. 
+
+### Use multiple accounts to separate testing and production environments [#environments]
+
+If you create multiple accounts you can observe data across multiple environments. With multiple accounts, you can monitor changes before they go to production. If you’re updating your application weekly, or even daily, it is important that you have a strong testing environment in place so you can make sure each deployment is smooth and error free.
```
Comment (akristen) [inline src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:53]:
```suggestion
Multiple accounts let you observe data across multiple environments so you can monitor changes before they go to production. If you’re updating your application regularly, it's important that you have a strong testing environment in place so you can make sure each deployment is error free.
```

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+
+<img
+    title="multiple environments"
+    alt="a diagram demonstrating how to use New Relic with multiple environments"
+    src={accountsMultipleAccountEnvironments}
+/>
+
+### Use multiple accounts to separate projects with unique data sets [#data-sets]
+
+If you manage a payment provider that works in multiple countries, you might need to manage different currencies and different tax regulations. Or, you could manage a social media site and you might need to comply with different data collection policies in different regions. In both those situations, it is a good idea to create multiple accounts.
```
Comment (akristen) [inline src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:64]: Can you unspool more literally what these examples have in common so people can extract how it applies to them if they don't fit these two examples. For example, maybe clarifying that multiple accounts is good if you have competing laws that need to be separated (idk how to word it but thats just me riffing)

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx (hunk 3 above)
Comment (akristen) [inline src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:39]: Something to think about, maybe you have one h2 that's 'Choose a single account structure' that encompasses these bullet points. Then "Choose multiple accounts" (or something) that has all your h3s.

**Review by akristen — CHANGES_REQUESTED** [review]
OK, this is a good breakdown on choosing account structure within an organization. I might consider updating the title to reflect that this is a guiding doc for helping you determine if you should have one account or more.

- On a copy + prose level, it’s good and well written. 
- My only writing-level feedback is to maybe break up some of the text so its chunkable + more readable. For example, a few paragraphs are back to back and have >4 sentences in a row. I think it's better to use bullet points esp because it's a concept-heavy doc. 
- I did leave some questions about some of the concepts. Nothing too new-writing heavy, but just suggestions on unpacking some of the elements.  

When you’re ready to merge it, LMK and i’ll approve

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx (hunk 1 above)
Comment (homelessbirds) [reply src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:28]: Done :)

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx (hunk 2 above)
Comment (homelessbirds) [reply src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:35]: Yes, @akristen! Good callout. Currently waiting on an SME to get back to me so I can improve that definition.

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx (hunk 4 above)
Comment (homelessbirds) [reply src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:49]: What if I say: If these apply to you, then you `might` only need one account. Otherwise, we recommend considering a multi-account structure to fit your needs.

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx (hunk 5 above)
Comment (homelessbirds) [reply src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:62]: that's a good callout, thanks!

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx (hunk 4 above)
Comment (akristen) [reply src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx]: Yes, I think that's a good halfway point. Esp since you can have a separate section about multiple accounts

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+
+This doc will guide you through setting up your account structure. If you're going to create multiple accounts, it is best to do that _before_ you begin ingesting data. It can be harder to make changes after you connect New Relic to your applications because you risk losing historical context.
+
+<Callout variant="tip">
+If you're using Free or Standard editions, you only have one account available and you won't need this doc. Feel free to revisit this page if you decide to change to [Pro or Enterprise](https://newrelic.com/pricing)
+</Callout>
+
+## Before you get started [#begin]
+
+In this guide we're going to help you create and manage your New Relic account structure.
```
Comment (rhetoric101) [inline src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:33]: This sentence seems more like one I'd expect in the introduction. Consider just having a sentence like this: Make sure you're familiar with some terms that are key to your decision to create additional accounts.

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+If you're using Free or Standard editions, you only have one account available and you won't need this doc. Feel free to revisit this page if you decide to change to [Pro or Enterprise](https://newrelic.com/pricing)
+</Callout>
+
+## Before you get started [#begin]
+
+In this guide we're going to help you create and manage your New Relic account structure. 
+
+First, make sure you're familiar with our terms:
+
+* **Organization**:  An organization represents a New Relic customer. Your New Relic organization contains your organization's accounts, users, and data.
```
Comment (rhetoric101) [inline src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:37]:
```suggestion
* **Organization**:  An organization represents a New Relic customer. Your New Relic organization contains accounts, users, and data.
```

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+* **User**: In your organization, your [New Relic users](/docs/accounts/accounts-billing/new-relic-one-user-management/introduction-managing-users/) are granted access to specific accounts that are relevant to their duties and responsibilities. To manage users' access to accounts, you grant groups access to specific roles on specific accounts. Our user management system allows you to create the user access you need, whether that's a relatively simple setup with just a few roles across a few accounts, or a complex one with many roles across many accounts. 
+
+## Single account structure [#single-account-structure]
+
+Signing up for New Relic automatically generates an organization. An organization contains one account unless you manually create more. One account can be perfect for many companies and it might not be necessary for you to add more. 
+
+Some indications that one account is right for you:
+
+* You have 200 employees or less and you don't anticipate growing too much
```
Comment (rhetoric101) [inline src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:47]: It seems like there could be many exceptions to the rule-of-thumb. Are there other factors that combine with the 200 number that makes this clearer?

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx [hunk 11]
```diff
@@ -0,0 +1,124 @@
...
+* You aren't ingesting vast amounts of data. If you're not sure, use [our spreadsheet](https://newrelic.com/blog/nerdlog/estimate-data-cost) to estimate the amount of data you might send to New Relic
```
Comment (rhetoric101) [inline src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:48]: Could you provide some examples of vast?  As a new users may have no idea what a large amount of data might look like in terms of GB.

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx [hunk 12]
```diff
@@ -0,0 +1,124 @@
...
+* You expect to send a moderate amount of alerts. If you're not sure, review our [alerting rules and limits](/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/rules-limits-alerts/)
```
Comment (rhetoric101) [inline src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:49]: What is a moderate amount? Could you maybe provide a range over a period of time?

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx [hunk 13]
```diff
@@ -0,0 +1,124 @@
...
+
+If these apply to you, then you might only need one account. Otherwise, we recommend considering a multi-account structure to fit your needs.
+
+## Create multiple accounts [#multiple-accounts]
+
+There are many reasons to create multiple accounts within your organization. Here are some common examples that can help you decide if a multi-account structure is right for you and your team.
+
+### Use multiple accounts to separate testing and production environments [#environments]
```
Comment (rhetoric101) [inline src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:57]: It seems like all software companies would be interested in monitoring each phase of the development lifecycle. Are we suggesting that all Pro and Enterprise edition organizations follow this? What are the benefits and drawbacks?

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx [hunk 14]
```diff
@@ -0,0 +1,124 @@
...
+    title="multiple environments"
+    alt="a diagram demonstrating how to use New Relic with multiple environments"
+    src={accountsMultipleAccountEnvironments}
+/>
+
+### Separate projects with unique data sets with multiple accounts [#data-sets]
+
+If you manage a payment provider that works in multiple countries, you might need to manage different currencies and different tax regulations. Or, you could manage a social media site and you might need to comply with different data collection policies in different regions. In both those situations, it is a good idea to create multiple accounts. 
+
+<img
```
Comment (rhetoric101) [inline src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:72]: This image might be a bit clearer if you represented the organization as a box (container) that holds the three accounts. In this diagram, it almost looks like the accounts are not contained within the organization.

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx [hunk 15]
```diff
@@ -0,0 +1,124 @@
...
+    style={{ align: 'left',maxWidth: '100%' }}
+    title="multiple data sets"
+    alt="A diagram showing how to use New Relic with multiple data sets"
+    src={accountsUniqueDataSets}
+/>
+
+### Use multiple accounts to manage large data limits [#large-data-limits]
+
+If you plan on ingesting vast amounts of data, it’s a good idea to create multiple accounts to avoid data limits. Financial institutions and healthcare providers often ingest high volumes of data because they track transactions and user information across multiple states and at high rates. Large retailers also benefit from creating multiple accounts. If you are a parent organization and you manage several, smaller companies, creating an account for each company could be a good idea.
```
Comment (rhetoric101) [inline src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:81]: The last sentence about parent organizations managing smaller companies, might need some clarification: If this is a genuine case where one parent organization manages other managed organizations, that's a case of multi-tenancy, which is under development. There is an older partner model where one organization uses accounts as a proxy for its subsidiary organizations, but I don't think we want to emphasize that.

**Review by rhetoric101 — COMMENTED** [review]
@homelessbirds this has some nice, clear information about accounts! I added some suggestions you might consider.

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx [hunk 16]
```diff
@@ -0,0 +1,124 @@
+---
+title: Create and manage your accounts
```
Comment (paperclypse) [inline src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:2]: I think we want something that it makes it more clear about what the goal is for this doc --> "Account setup best practices"

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx [hunk 17]
```diff
@@ -0,0 +1,124 @@
...
+
+import accountsManageSecurity from 'images/accounts_diagram_manage-security.webp'
+
+import accountsMultipleAccountsforLargeDataLimits from 'images/accounts_diagram_multiple-accounts-for-large-data-limits.webp'
+
+import accountsUniqueDataSets from 'images/accounts_diagram_unique-data-sets.webp'
+
+import accountsMultipleAccountEnvironments from 'images/accounts_diagram_multiple-account-environments.webp'
+
+Before you get started with New Relic, it's important to take a moment to think about how you want to use the data you're about to send to New Relic. Do you manage a parent organization with a few different subsidiary companies? Are you a small organization that's growing fast and you need to monitor your application at every stage of the development process?  You can easily customize New Relic to meet your organization's specific needs using accounts.
```
[the remark this answers is not in the document]
Comment (paperclypse) [reply src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:16]: Multi-tenancy is definitely outside the scope of this doc.  Perhaps the use of the word "organization" in this intro is confusing the issue?

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx [hunk 18]
```diff
@@ -0,0 +1,124 @@
...
+import accountsUniqueDataSets from 'images/accounts_diagram_unique-data-sets.webp'
+
+import accountsMultipleAccountEnvironments from 'images/accounts_diagram_multiple-account-environments.webp'
+
+Before you get started with New Relic, it's important to take a moment to think about how you want to use the data you're about to send to New Relic. Do you manage a parent organization with a few different subsidiary companies? Are you a small organization that's growing fast and you need to monitor your application at every stage of the development process?  You can easily customize New Relic to meet your organization's specific needs using accounts. 
+
+With New Relic's account structure you can:
+
+* Maximize data ingest
+* Keep data organized
```
[the remark this answers is not in the document]
Comment (paperclypse) [reply src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:21]: Probably. I think I'd prefer some other word than "segregated" though

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx (hunk 11 above)
Comment (paperclypse) [reply src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:48]: I agree that it's good to get specific here.

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx [hunk 19]
```diff
@@ -0,0 +1,124 @@
...
+With New Relic's account structure you can:
+
+* Maximize data ingest
+* Keep data organized 
+* Monitor multiple environments, like staging and development
+* Manage billing across different areas of your organization
+
+This doc will guide you through setting up your account structure. If you're going to create multiple accounts, it is best to do that _before_ you begin ingesting data. It can be harder to make changes after you connect New Relic to your applications because you risk losing historical context.
+
+<Callout variant="tip">
```
[the remark this answers is not in the document]
Comment (homelessbirds) [reply src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:27]: Thanks!

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx (hunk 16 above)
Comment (homelessbirds) [reply src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:2]: Good idea!

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx (hunk 15 above)
Comment (homelessbirds) [reply src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:81]: I changed the language here to make it more clear!

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx (hunk 14 above)
Comment (homelessbirds) [reply src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:72]: I think I fixed this language too. Good callout, Rob.

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx [hunk 20]
```diff
@@ -0,0 +1,138 @@
...
+<img
+    style={{ align: 'left',maxWidth: '100%' }}
+    title="multiple environments"
+    alt="a diagram demonstrating how to use New Relic with multiple environments"
+    src={accountsMultipleAccountEnvironments}
+/>
+
+### Separate projects with unique data sets with multiple accounts [#data-sets]
+
+An organization is a container that holds different data for different accounts. Accounts can be a good way to keep data separate so that you can easily filter and query based on specific requirements. For example, if you manage a payment provider that works in multiple countries, you might need to manage different currencies and different tax regulations and you don't want the data from each country to mix with the others. Or, you could manage a social media site and you might need to comply with different data collection policies in different regions. In both those situations, it is a good idea to create multiple accounts.
```
[the remark this answers is not in the document]
Comment (ally-sassman) [reply src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:67]: Just a restructuring suggestion!

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx [hunk 21]
```diff
@@ -0,0 +1,138 @@
...
+
+Some indications that one account is right for you:
+* You aren't ingesting vast amounts of data. If you're not sure, use [our spreadsheet](https://newrelic.com/blog/nerdlog/estimate-data-cost) to estimate the amount of data you might send to New Relic
+* You expect to send a moderate amount of alerts. If you're not sure, review our [alerting rules and limits](/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/rules-limits-alerts/)
+
+If these apply to you, then you might only need one account. Otherwise, we recommend considering a multi-account structure to fit your needs.
+
+## Create multiple accounts [#multiple-accounts]
+
+There are many reasons to create multiple accounts within your organization. Here are some common examples that can help you decide if a multi-account structure is right for you and your team.
```
[the remark this answers is not in the document]
Comment (ally-sassman) [reply src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:52]: To simplify the long sentence

**Review by ally-sassman — DISMISSED** [review]

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx [hunk 22]
```diff
@@ -0,0 +1,138 @@
...
+* Alert conditions default to 4,000 conditions per account.
+* Alert workflows default to 4,000 workflows per account.
+* Alert policies default 10,000 policies per account.
+* Alert destinations default 2,000 destinations per account. 
+* NRDB query inspected count limits are different with standard and data plus accounts
+
+To learn more about limits on inspected data points, see our [NRQL limit docs](/docs/query-your-data/nrql-new-relic-query-language/get-started/rate-limits-nrql-queries/#inspected-count-limits). And to learn more about alerting limits, see [our documentation](docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/rules-limits-alerts/).
+
+To learn more, see our 
+- # of Queries per Minute 3000/account/1 min
```
[the remark this answers is not in the document]
Comment (homelessbirds) [reply src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:97]: Yes! This was. Thanks for catching

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx [hunk 23]
```diff
@@ -0,0 +1,142 @@
...
+
+* Alert conditions default to 4,000 conditions per account.
+* Alert workflows default to 4,000 workflows per account.
+* Alert policies default 10,000 policies per account.
+* Alert destinations default 2,000 destinations per account. 
+* NRDB query inspected count limits are different with standard and data plus accounts.
+
+To learn more about limits, see:
+*  [Inspected data points for NRQL queries](/docs/query-your-data/nrql-new-relic-query-language/get-started/rate-limits-nrql-queries/#inspected-count-limits)
+* [Alerting limits](docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/rules-limits-alerts/).
```
Comment (akristen) [inline src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx:94]: Nitpicky but as a fast follow I'd remove this `.`

**Review by akristen — APPROVED** [review]

File: src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx (hunk 23 above)
Comment (homelessbirds) [reply src/content/docs/accounts/accounts-billing/account-structure/create-manage-accounts.mdx]: Fixed :)

**Comment by homelessbirds** [comment]
Merging because of[ this build](https://www.gatsbyjs.com/dashboard/34e708d2-68a1-48ca-b66f-d2604890b061/sites/a0b77314-cdea-4790-8fde-7d3457d590ce/builds/ac016035-6985-4f85-b353-19d79ea3bebe/details?loggedin=1&token=eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiJhZTUzYjNkZS1jNzFmLTRlZjItODczYS1kNDk2NDU3ZDFkMjgiLCJlbWFpbCI6InNkb3duaW5nQG5ld3JlbGljLmNvbSIsImlhdCI6MTcwNDgxOTk0NSwiZXhwIjoxNzA0ODYzMTQ1LCJpc3MiOiJHYXRzYnkgSW5jIiwic3ViIjoic2Rvd25pbmdAbmV3cmVsaWMuY29tIn0.KZ9aPpUSZEuzCzkbNSzfXwMFEsjVJ10zEaz4TpRkgqcICEgbOJcJFESzfaZ64Bm7zIu8vb1d-FNqXIsCVL3IPvy987vn5_XH2KcD4_2rwyjx-_4suGMuaH9NlRCNG17ohoCoqKkAfUDFCD5CkwGAd_Sxgxi2Xr1TKeqlavEPRQLDG4PluHcyTyrA1_djWu6IreMsxpUY6QPutIACvdscoLOSF1lFvzOr4ZvRQvAJZGSDgoT-IczzaRCRNsgOCsV1pgWVq3VmdKFJhzA8FAt8l1nGNjv_rr1p-cllI5NGO3yV1z4pgETbDozwxdLxdRjO0kz_JuhqUuvWSVi7AhBSIJ1juv0jcFGC-7ZhZp9iFBfw_5iv7Ejp8sG5ts-Q7afl1AMoWa8JQg1Hma4tAzmUW6aX4O60tQqjHEMbraTm63sV9GMC2RZ3RRvQBJyqqvTCZdJjEwlQPGOOBqz4MQjh3NxPw3bEk4mq99EcQXDXCqbZOY4qXMxtbn4OJEJZG5umrCkHVAAXVeqF2f_WDWddNYWejAceTX54VU4Hp13dSnFWpX46h3C6-QgSFBBIHG6YFe3y1sbklhirU_u9WODtlyLSeAp6oWDkJeUpK8pLJgB3ERNcDZKuKz5Jncg27c7tgfQiEtazjggKfbBTcFx6xVemkHcCu05SC6BjL2l_OLI&expiration=2024-01-23T17%3A05%3A45.430Z&ssoToken=&ssoExpiration=&id=ae53b3de-c71f-4ef2-873a-d496457d1d28&loginRedirectUrl=%2Fdashboard%2F34e708d2-68a1-48ca-b66f-d2604890b061%2Fsites%2Fa0b77314-cdea-4790-8fde-7d3457d590ce%2Fbuilds%2Fac016035-6985-4f85-b353-19d79ea3bebe%2Fdetails&selectedPlan=&redirectToLeadCapture=false)
